### DISCUSSION TITLE
Undocumented Street View source for official coverage: "innerspace"

### DISCUSSION THREAD

**Discussion by Miss-Inputs** [sk-zk/streetlevel#3] [General]
Trying to figure out what this is exactly so it can be added to the documentation.

Stumbled across some panos that are uploaded by Google and are inside buildings, but are not "scout" like you would expect.

zCPgPH5wkquvQHJ5hQNORg - a local pub, my first thought was that the Google employee wanted some drinks after a hard day of going around taking photos of things. But maybe not? There is a pano that is technically outside that is part of this (zNekcWLqU6sZSY1NOs9_OQ) that is still innerspace, probably because it's part of the same set of panos.

I wanted to know how it's captured, luckily mirrors exist in real life, and the City Gym in Sydney [(bGNF2YXUaRTdT1h8CNC20w)](https://www.google.com/maps/@-33.8749195,151.2157491,2a,15y,357.92h,88.84t/data=!3m6!1e1!3m4!1sbGNF2YXUaRTdT1h8CNC20w!2e0!7i13312!8i6656?entry=ttu) also happens to have source = "innerspace". So it is just a tripod. Cool!

Otherwise, they seem to behave identically to scout. I don't think it's exclusive to Australia, that's just where I've looked.

If I had to guess, scout has more to do with the Arts & Culture thing, and innerspace is just advertisements for local businesses or something?

Might not be relevant to this project at all, but it interested me, and I probably wouldn't have noticed if not for this.

**Comment by sk-zk** [comment]
I think it's this: https://en.wikipedia.org/wiki/Street_View_Trusted
